app/assets/config/spree_videos_manifest.js

```diff
@@ -0,0 +1,129 @@
+//= link_tree ../images
+
+//= link spree/backend/all.js
```

Comment:
@adhik-mehta the manifest can't be simply copy pasted, these commented lines mean something. none of these files exist in spree_videos repository so they cant be included in the manifest